packages/popover/components/PopoverListItem.tsx

```diff
@@ -84,33 +84,19 @@ const PopoverListItem = (props: PopoverListItemProps) => {
           itemContent: children
         };
       }
-      const isItemGraphic = child =>
-        child.type === PopoverListItemIcon ||
+      const isItemGraphic = (
+        child
+      ): child is typeof PopoverListItemIcon | typeof PopoverListItemAvatar =>
+        (React.isValidElement(child) && child.type === PopoverListItemIcon) ||
         child.type === PopoverListItemAvatar;
+
       const itemGraphicStart = itemChildren.find(
-        child =>
-          React.isValidElement<PopoverListItemIcon | PopoverListItemAvatar>(
-            child
-          ) &&
-          isItemGraphic(child) &&
-          child.props.position === "start"
+        child => isItemGraphic(child) && child.props.position === "start"
       );
       const itemGraphicEnd = itemChildren.find(
-        child =>
-          React.isValidElement<PopoverListItemIcon | PopoverListItemAvatar>(
```

Comment:
Why did you decide to drop the `isValidElement`?

Reply:
It's not dropped, it's moved - https://github.com/dcos-labs/ui-kit/pull/479/files#diff-cac05b4db96f03850867c34aed16603aR90 .